install

```diff
@@ -33,6 +43,7 @@ This script installs the bundled theme styles for gedit.
 OPTIONS:
   -h    Shows this message
   -a    Install for all users
+  -o	Old version support (Gedit 2.x version)
```

Comment:
Do you mind changing this tab character separating `-o` from `Old...` with 4 spaces instead? Otherwise it could look wrong with different tab width settings. Other than that everything looks good to me.